wiki/content/deploy/dgraph-zero.md

```diff
@@ -62,121 +83,201 @@ The `/state` endpoint of Dgraph Zero returns a JSON document of the current grou
+          "groupId": 1,
+          "predicate": "dgraph.graphql.schema_created_at",
+          "force": false,
+          "space": "0",
+          "remove": false,
+          "readOnly": false,
+          "moveTs": "0"
+        },
+        "dgraph.graphql.schema_history": {
           "groupId": 1,
-          "predicate": "counter.val"
+          "predicate": "dgraph.graphql.schema_history",
+          "force": false,
+          "space": "0",
+          "remove": false,
+          "readOnly": false,
+          "moveTs": "0"
+        },
+        "dgraph.graphql.xid": {
+          "groupId": 1,
+          "predicate": "dgraph.graphql.xid",
+          "force": false,
+          "space": "0",
+          "remove": false,
+          "readOnly": false,
+          "moveTs": "0"
         },
         "dgraph.type": {
           "groupId": 1,
-          "predicate": "dgraph.type"
+          "predicate": "dgraph.type",
+          "force": false,
+          "space": "0",
+          "remove": false,
+          "readOnly": false,
+          "moveTs": "0"
         }
       },
-      "checksum": "1021598189643258447"
+      "snapshotTs": "22",
+      "checksum": "18099480229465877561"
     }
   },
   "zeros": {
     "1": {
       "id": "1",
+      "groupId": 0,
       "addr": "zero1:5080",
-      "leader": true
+      "leader": true,
+      "amDead": false,
+      "lastUpdate": "0",
+      "clusterInfoOnly": false,
+      "forceGroupId": false
     },
     "2": {
       "id": "2",
-      "addr": "zero2:5080"
+      "groupId": 0,
+      "addr": "zero2:5082",
+      "leader": false,
+      "amDead": false,
+      "lastUpdate": "0",
+      "clusterInfoOnly": false,
+      "forceGroupId": false
     },
     "3": {
       "id": "3",
-      "addr": "zero3:5080"
+      "groupId": 0,
+      "addr": "zero3:5083",
+      "leader": false,
+      "amDead": false,
+      "lastUpdate": "0",
+      "clusterInfoOnly": false,
+      "forceGroupId": false
     }
   },
   "maxLeaseId": "10000",
   "maxTxnTs": "10000",
-  "cid": "3602537a-ee49-43cb-9792-c766eea683dc",
+  "maxRaftId": "3",
+  "removed": [],
+  "cid": "2571d268-b574-41fa-ae5e-a6f8da175d6d",
   "license": {
+    "user": "",
     "maxNodes": "18446744073709551615",
-    "expiryTs": "1578704367",
+    "expiryTs": "1605942487",
     "enabled": true
   }
 }
 ```
 
-Here’s the information the above JSON document provides:
+This JSON provides information that includes the following:
 
-- Group 0
-  - members
-    - zero1:5080, id: 1, leader
-    - zero2:5080, id: 2
-    - zero3:5080, id: 3
 - Group 1
-    - members
-        - alpha1:7080, id: 1, leader
-        - alpha2:7080, id: 2
-        - alpha3:7080, id: 3
-    - predicates
-        - dgraph.type
-        - counter.val
+  - Members (HTTP port number is shown after each node name):
+    - alpha2:7082, id: 1, leader
+    - alpha1:7080, id: 2
+    - alpha3:7083, id: 3
+    - zero1:5080, id: 1, leader
+    - zero2:5082, id: 2
+    - zero3:5083, id: 3
```

Comment:
Alphas belong to group 1 and Zeros belong to group 2. We can follow the same format as earlier.